oauth-server/src/main/java/io/strimzi/kafka/oauth/server/OAuthJWTAuthorizer.java

```diff
@@ -0,0 +1,68 @@
+package io.strimzi.kafka.oauth.server;
+
+import kafka.network.RequestChannel;
+import kafka.security.auth.Acl;
+import kafka.security.auth.Authorizer;
+import kafka.security.auth.Operation;
+import kafka.security.auth.Resource;
+import org.apache.kafka.common.security.auth.KafkaPrincipal;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scala.collection.immutable.Map;
+import scala.collection.immutable.Set;
+
+public class OAuthJWTAuthorizer implements Authorizer {
```

Comment:
I think since authz is not support it's best not to include this class in the PR.